### PR TITLE
[#487] Added support for attachments to Stuff ZDS registration backend

### PR DESCRIPTION
**Pull request by Bartvaderkin**

### CODE REVIEW

File: src/openforms/registrations/contrib/stuf_zds/client.py [hunk 1]
```diff
@@ -287,6 +287,43 @@ def create_zaak_document(
...
+                "zaak_identificatie": zaak_id,
+                "document_identificatie": doc_id,
+                "titel": "bijlage",
+                "auteur": "open-forms",
+                "taal": "nld",
+                "inhoud": base64_body,
+                "status": "definitief",
+                "bestandsnaam": submission_attachment.get_display_name(),
+                "formaat": submission_attachment.get_format(),
+                "beschrijving": "Bijgevoegd document",
```
Comment (sergei-maertens) [inline src/openforms/registrations/contrib/stuf_zds/client.py:314]: Fine for now, but I expect in the future there will be meta-data on the form field that shall serve as input for the attributes here, such as the label of the form field etc.

File: src/openforms/submissions/models.py [hunk 2]
```diff
@@ -560,3 +560,6 @@ def delete(self, using=None, keep_parents=False):
 
     def get_display_name(self):
         return self.file_name or self.original_name
+
+    def get_format(self):
+        return os.path.splitext(self.get_display_name())[1].lstrip(".")
```
Comment (sergei-maertens) [inline src/openforms/submissions/models.py:565]: Shouldn't this be a mime type? e.g. `application/pdf`

**Review by sergei-maertens — COMMENTED** [review]
One question about the format

File: src/openforms/submissions/models.py (hunk 2 above)
Comment (Bartvaderkin) [reply src/openforms/submissions/models.py:565]: All examples are extensions but the spec indeed prefers mime type so I modified the PR.

**Review by sergei-maertens — APPROVED** [review]

**Pull request merged**